### PR TITLE
Added Sample Data For AgendaCategory & Removed Delete Fund/Campaign Mutations

### LINKED ISSUE #2510: Add Sample Data for AgendaCategory & Remove Mutations for Deleting Funds/Campaigns
**Is your feature request related to a problem? Please describe.**
Like Sample data for Action Item Category we need sample Agenda Categories for better UX in Admin. Mutations for removing Funds and Campaigns should be removed as only disable function should be present for these.

**Describe the solution you'd like**
- Add sample data for Agenda Category
- Remove `RemoveFund` mutation.
- Remove `RemoveFundraisingCampaign` mutation.

**Additional context**
Above suggestion were discussed in past GSoC bi-weekly meet

### PR DESCRIPTION
**Pull request by GlenDsza**
**What kind of change does this PR introduce?**

refactoring

**Issue Number:**

Fixes #2510 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

Default agendaCategories being created on importing sample data

image

**Summary**

- Added Sample Data (Default category) for `AgendaCategories` for each organization
- Removed `RemoveFund `Mutation
- Removed `RemoveFundraisingCampaign` Mutation

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes 

## Summary by CodeRabbit

- **New Features**
  - Introduced a new JSON file for action item categories, providing default templates for better task organization.

- **Bug Fixes**
  - Removed outdated mutation operations for removing funds and fundraising campaigns, streamlining API functionality.

- **Chores**
  - Enhanced data handling capabilities by integrating support for agenda categories in the database operations.

- **Tests**
  - Added unit tests for error handling related to input validation and invalid file types, ensuring robustness in error management.
  - Introduced tests for the `forgotPassword` mutation to validate handling of invalid OTP scenarios.

### CODE REVIEW

File: src/utilities/loadSampleData.ts [hunk 1]
```diff
@@ -120,6 +122,9 @@ async function insertCollections(collections: string[]): Promise<void> {
         case "actionItemCategories":
           await ActionItemCategory.insertMany(docs);
           break;
+        case "agendaCategories":
+          await AgendaCategoryModel.insertMany(docs);
+          break;
```
[the remark this answers is not in the document]
Comment (GlenDsza) [reply src/utilities/loadSampleData.ts:127]: Later batch writes for all sample data insertion and formatting could be introduced across sample data categories

File: src/resolvers/Mutation/forgotPassword.ts [hunk 2]
```diff
@@ -27,8 +27,8 @@ export const forgotPassword: MutationResolvers["forgotPassword"] = async (
   const { userOtp, newPassword, otpToken } = args.data;
 
   try {
-    await jwt.verify(otpToken, ACCESS_TOKEN_SECRET as string);
-  } catch (error) {
+    jwt.verify(otpToken, ACCESS_TOKEN_SECRET as string);
+  } catch {
```
[the remark this answers is not in the document]
Comment (GlenDsza) [reply src/resolvers/Mutation/forgotPassword.ts:31]: verify function doesn't return a promise

File: tests/resolvers/Mutation/forgotPassword.spec.ts [hunk 3]
```diff
@@ -125,6 +125,40 @@ describe("resolvers -> Mutation -> forgotPassword", () => {
...
+    };
+
+    try {
+      await forgotPasswordResolver?.({}, args, {});
+    } catch (error: unknown) {
+      if (error instanceof Error) {
+        expect(error.message).toEqual(INVALID_OTP);
+      }
+    }
+  });
```
[the remark this answers is not in the document]
Comment (GlenDsza) [reply tests/resolvers/Mutation/forgotPassword.spec.ts:160]: is already quite readable and efficient

**Comment by GlenDsza** [comment]
@tasneemkoushar PTAL

**Review by tasneemkoushar — APPROVED** [review]

**Pull request merged**